pkg/linksharing/web/static/css/style.css

```diff
@@ -116,19 +116,33 @@ p, span, a {
 /* Content */
 
 .sidebar {
+  position: relative;
   border-left: 1px solid #EDEEEF;
   overflow-x: hidden;
   transition: all 400ms ease;
 }
 
-.expander {
-  width: 20px;
-  height: 20px;
+.expander,
+.collapser {
+  width: 26px;
+  height: 26px;
   cursor: pointer;
+  position: absolute;
+  top: 50vh;
+  transform: translateY(-50%);
```

Comment:
This need for translateY looks weird. Any reason you are not using `top: calc(50vh - 13px)`?

Or better yet flexbox for aligning things?

Reply:
transform: translateY(-50%) needs to not hardcode height/2, so if expander block parameters will be changed we will not change css in 2 places.

Reply:
And flexbox?

Reply:
can not see any point for creating 2 more containers to use flexbox for centering 1 image

Reply:
it will also cause padding problems